src/tools/miri/tests/pass/dyn-upcast.rs

```diff
@@ -482,3 +484,51 @@ fn drop_principal() {
     println!("before");
     drop(y);
 }
+
+fn modulo_binder() {
```

Comment:
Might be worth adding links to the respective issues? (Also applies to the rustc ui tests.)